java-lib/src/main/java/com/wavefront/ingester/AbstractIngesterFormatter.java

```diff
@@ -674,12 +676,88 @@ public static List<String> getDefaultLogMessageKeys() {
     return DEFAULT_LOG_MESSAGE_KEYS;
   }
 
+  @Nullable
+  public static String getLogLevel(@Nullable List<Annotation> annotations,
+                                     @Nullable List<String> customLogLevelTags) {
+    String logLvlStr = null;
+    if (annotations != null) {
+      Iterator<Annotation> iter = annotations.iterator();
+      while (iter.hasNext()) {
+        Annotation annotation = iter.next();
+        for (String defaultLogServiceKey : DEFAULT_LOG_LEVEL_KEYS) {
```

Comment:
nit: for clarity, should this variable be `defaultLogLevelKey`?

Reply:
Done